go-controller/pkg/ovn/namespace.go

```diff
@@ -112,22 +112,43 @@ func createIPAddressSlice(ips []*net.IPNet) []net.IP {
 // Traffic will be dropped by the default multicast deny ACL.
 func (oc *Controller) multicastUpdateNamespace(ns *kapi.Namespace, nsInfo *namespaceInfo) {
 	if !oc.multicastSupport {
+		klog.V(5).Infof("Multicast is not supported")
 		return
 	}
 
 	enabled := (ns.Annotations[nsMulticastAnnotation] == "true")
 	enabledOld := nsInfo.multicastEnabled
 
 	if enabledOld == enabled {
+		klog.V(5).Infof("Multicast state did not change")
 		return
 	}
 
 	var err error
 	nsInfo.multicastEnabled = enabled
 	if enabled {
-		err = oc.createMulticastAllowPolicy(ns.Name, nsInfo)
+		klog.V(5).Infof("Multicast is enabled for Namspace %s and the port group is being updated", ns.Name)
+		err = nsInfo.updateNamespacePortGroup(ns.Name)
+		if err != nil {
+			klog.Errorf(err.Error())
```

Comment:
explanatory messages are nice for troubleshooting 
```suggestion
			klog.Errorf("Error trying to update namespace %s: %v", ns.Name, err))
```

Reply:
I completely agree thanks :)

Reply:
I realised later that is allover the code, but if you can change at least the ones that you are changing it will be great.

Reply:
Yeah we need to start somewhere, for debugging I had to add a bunch of print statements